lib/roadmap.rb

```diff
@@ -0,0 +1,13 @@
+module ROADMAP
```

Comment:
It's convention to titleize modules or class names. Prefer `Roadmap` over `ROADMAP`.